docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -379,12 +408,13 @@ data:
     appender.rolling.strategy.type = DefaultRolloverStrategy
     appender.rolling.strategy.max = 10
 
-    # Suppress the irrelevant (wrong) warnings from the Netty channel handler
+    # 关闭 Netty channel handler 中的不相关（错误）警告
```

Comment:
here, too.

Reply:
Ok, take your advice. It has been fixed.